apps/heft/src/plugins/NodeServicePlugin.ts

```diff
@@ -125,7 +125,8 @@ export class NodeServicePlugin implements IHeftPlugin {
           });
 
           build.hooks.compile.tap(PLUGIN_NAME, (compile: ICompileSubstage) => {
-            compile.hooks.afterEachIteration.tap(PLUGIN_NAME, this._compileHooks_afterEachIteration);
+            compile.hooks.afterCompile.tap(PLUGIN_NAME, this._compileHooks_afterEachCompile);
+            compile.hooks.afterRecompile.tap(PLUGIN_NAME, this._compileHooks_afterEachCompile);
```

Comment:
`afterRecompile` is supposed to be an async hook.  However the function it is calling here is a synchronous function.

Is that okay?  If not, is there some way that Heft could replace the tapable typings with a stricter declaration?

Reply:
Yeah a sync tap is fine.